### ISSUE TITLE
Optical flow training tricks

### ISSUE THREAD

**Issue by qijiezhao** [yjxiong/tsn-pytorch#15]
Hi, Xiong:

sorry to impose again, My experiment of optical flow only receives an average accuracy of 84.4% on the split 1( result is given by the test script). My training setting is (batch_size=128, init_lr=0.001, lr_step=(190,300 epochs)), which is different from the caffe implement version of TSN (batch_size=24, init_lr=0.005,lr_step=(10000,16000 iterations)).

can you please help figure out which setting can bring the proposed result on your paper? 
If both of them can't, so what is the best setting?

**Comment by yjxiong** [comment]
This repo is now in experimental release. Please follow the settings in

https://github.com/yjxiong/temporal-segment-networks/blob/master/models/ucf101/tsn_bn_inception_flow_solver.prototxt

**Comment by qijiezhao** [comment]
thanks again for your kindly reply!
